docs/basics/101-113-symlinks.rst

```diff
@@ -0,0 +1,222 @@
+
+There is one detail about DataLad datasets that we haven't covered yet. Its both
+a crucial aspect to understanding certain aspects of a dataset, but it is also a
+potential source of confusion that we want to eradicate.
+
+You might have noticed already that an ``ls -l`` or ``tree`` command in your dataset shows small
+arrows and quite cryptic paths following each non-textfile. Maybe your shell also
+displays these files in a different color than textfiles when listing
+them. We'll take a look together, using the ``books/`` directory as an example:
+
+.. runrecord:: _examples/DL-101-113-101
+   :language: console
+   :workdir: dl-101/DataLad-101
+
+   # in the root of DataLad-101
+   $ cd books
+   $ tree
+
+If you don't know what you are looking at,
+this looks weird, if not worse: intimidating, wrong, or broken.
+First of all: no, **it is all fine**. But lets start with the basics of what is displayed
+here to understand it.
+
+The small ``->`` symbol connecting one path (the books name) to another path (the weird
+sequence of characters ending in ``.pdf``) is what is called a
+*symbolic link* (short: :term:`symlink`) or *softlink*.
+It is a term for any file that contains a reference to another file or directory as
+a :term:`relative path` or :term:`absolute path`.
+If you use Windows, you are familiar with a related concept: a shortcut.
+
+This means that the files that are in the locations in which you saved content
+to and are named as you named your files (e.g. ``TLCL.pdf``),
+do *not actually contain your files' content*:
+they just point to the place where the actual file content secretly resides.
+
+This sounds weird, and like an unnecessary complication of things. But we will
+get to why this is relevant and useful shortly. First, however,
+where exactly are the contents of the files you created or saved?
+
+The start of the link path is ``../.git``. The section :ref:`createDS` contained
+a note that strongly advised that you to not temper with
+(or in the worst case, delete) the ``.git``
+repository in the root of any dataset. One reason
+why you should not do this is because *this* ``.git`` directory is where all of your file content
+is actually stored.
+
+But why is that? We have to talk a bit Git-annex now in order to understand it.
+Note, though, that the information below applies to everything that is not an
+*adjusted branch* in a Git-annex *v7 repository* -- this information does not make
+sense yet, but it will be an important reference point later on.
+Just for the record: Currently, we do not yet have a v7 repository
+in ``DataLad-101``, and the explanation below applies to our current dataset.
+
+When a file is saved into a dataset to be tracked, DataLad gives this file to Git-annex
+if it
+
+#. is larger than a certain (configurable) size
+
+#. and/or not of certain specified types (e.g. textfiles, as with the
+   ``text2git`` configuration template).
```

Comment:
From https://git-annex.branchable.com/tips/largefiles/

```
Normally commands like git annex add always add files to the annex. And when using the
v7 repository mode, even git add and git commit -a will add files to the annex.
```

ATM DataLad does not alter this configuration in general:

```
(datalad3-dev) mih@meiner /tmp % dl create check
[INFO   ] Creating a new annex repo at /tmp/check 
create(ok): /tmp/check (dataset)                                                                                             
(datalad3-dev) mih@meiner /tmp % cat check/.gitattributes

* annex.backend=MD5E
**/.git* annex.largefiles=nothing%                                                                                           
```

... unless it is instructed to.